tests/unit/lib/kwlib_test.sh

```diff
@@ -22,6 +22,11 @@ function setupPatch()
   cp -f tests/unit/samples/test.patch "$SHUNIT_TMPDIR"
 }
 
+function setupCoverLetter()
+{
+  cp -f tests/unit/samples/patch_files/0000-cover-letter.patch "$SHUNIT_TMPDIR"
```

Comment:
- Remember: https://kworkflow.org/content/codingstyle.html#avoid-short-options
- Add quotes around the path.
- I think you did not add this sample file.